prime-angular-frontend/src/app/modules/enrolment/pages/care-setting/care-setting.component.ts

```diff
@@ -106,7 +106,7 @@ export class CareSettingComponent extends BaseEnrolmentProfilePage implements On
 
   public filterCareSettingTypes(careSetting: FormGroup) {
     // Create a list of filtered care settings
-    if (this.careSettings.length) {
+    if (this.careSettings?.length) {
```

Comment:
Similarly, I suspect this might be a similar issue, as well, as the jobs on line 180.